packages/net/writebuffer.pony

```diff
@@ -0,0 +1,155 @@
+use "collections"
+
+class WriteBuffer
+  """
+  * A List[ByteSeq] maintained as WriteBuffer.
```

Comment:
@SeanTAllen I promise to update the documentation as this PR takes a better shape. I am assuming the questions you have are rhetorical to get me to document the intent of adding this file, and I completely understand that request.

Outside the above, just to give you a context, this came up from [https://github.com/ponylang/ponyc/issues/380](url) and [https://pony.groups.io/g/user/message/207].

All I am really doing in the example case is that I am sending some strings followed by a few pseudo random numbers over a UDP connection to someone. The goal of WriteBuffer is to make it easy for somebody to serialize data and hold that data in say packets/frames before sending it over to a file or network.